modules/test/playwright/tests/frontend-data-set-views-web/dateRangeFilters.spec.ts

```diff
@@ -121,24 +339,17 @@ fragmentTest(
 		});
 
 		const activeFilterButton = fdsFragmentPage.page.getByRole('button', {
-			name: `${filterLabel}: 1/1/2020 - 1/2/3020`,
+			name: `${filterLabel}:`,
```

Comment:
iirc I did this because a partial match is enough, which gives us some more freedom in case we want to change dates or output date format changes. In my understanding we are not testing button text content but locating the right button to click (this is debatable though)

Reply:
Either is fine really, it's a very minor tradeoff :+1: